contents/blog/experiments.md

```diff
@@ -0,0 +1,94 @@
+---
+date: 2022-03-21
+title: What launching Experimentation taught us about running effective A/B tests
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["neil-kakkar"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+---
+
+We just launched our Experimentation suite, and there's a ton we learned about running successful experiments.
+
+It was a no brainer product decision: Since you're already analysing your data in PostHog, and you're already using feature flags to roll out new features, why not have the capability to test how well these features are doing?
```

Comment:
```suggestion
It was a no brainer product decision: Since you're already analysing your data in PostHog, and you're already using [Feature Flags](/product/feature-flags) to roll out new features, why not have the capability to test how well these features are doing?
```